advocacy_docs/edb-postgres-ai/analytics/how_to_lakehouse_sync.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Lakehouse Sync
+navTitle: Lakehouse Sync
+description: How to perform a Lakehouse Sync.
+deepToC: true
+---
+
+## Overview
+
+Performing a Lakehouse Sync is a way to capture information from a transactional database at a point in time and sync that information to a Managed Store Location (MSL).
+
+The Lakehouse sync process organizes the transactional database data into Lakehouse tables stored in the MSL. This process allows the data to be queried by a Lakehouse node, which is optimized for higher-performance queries using a vectorized query engine designed for Lakehouse tables.
+
+## Performing a Lakehouse Sync
+
+### Prerequisites
+
+- a Postgres cluster hosted and managed by EDB Postgres AI® Cloud Service
```

Comment:
I thing it is Services, but I am not sure how we are using the product name. 
```suggestion
- a Postgres cluster hosted and managed by EDB Postgres AI® Cloud Services
```

Reply:
Looks like it is Cloud Service from this: https://www.enterprisedb.com/products/edb-postgres-ai-cloud-service